examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -121,14 +125,16 @@ sed -i s/\<NTASKS\>/$ranks/g run.daint.slurm
 sed -i s/\<NTASKSPERNODE\>/1/g run.daint.slurm
 sed -i s/\<CPUSPERTASK\>/$nthreads/g run.daint.slurm
 sed -i s/--output=\<OUTFILE\>/--hint=nomultithread/g run.daint.slurm
-sed -i s/00:45:00/00:30:00/g run.daint.slurm
+sed -i s/00:45:00/00:45:00/g run.daint.slurm
```

Comment:
Is this line now redundant?

Reply:
Yes, I retained it for possible future changes to be easier.